templates/shared_services/firewall/terraform/firewall.tf

```diff
@@ -223,7 +223,8 @@ resource "azurerm_firewall_network_rule_collection" "resource_processor_subnet"
       "AzureResourceManager",
       "AzureContainerRegistry",
       "Storage",
-      "AzureKeyVault"
+      "AzureKeyVault",
+      "AppService"
```

Comment:
Let's create this by using a pipeline step and not directly here. I'll show you...

Reply:
Done in the notifier's tf